event/listener.php

```diff
@@ -203,6 +203,25 @@ public function show_idea($event)
 			$this->auth->acl_get('f_vote', (int) $this->config['ideas_forum_id']) &&
 			$event['topic_data']['topic_status'] != ITEM_LOCKED);
 
+		$s_voted_up = $s_voted_down = false;
+		if ($idea['idea_votes_up'] || $idea['idea_votes_down'])
```

Comment:
Maybe also check the `$can_vote` status here so there’s no need to even go in here.